__tests__/ChannelPreviewMessenger-test.js

```diff
@@ -0,0 +1,103 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+import React from 'react';
+import { cleanup, render, fireEvent } from '@testing-library/react';
+import { ChannelPreviewMessenger } from '../src/components/ChannelPreviewMessenger';
+import uuidv4 from 'uuid/v4';
+import { getTestClient, createUserToken } from './utils';
+
+// Note: running cleanup afterEach is done automatically for you in @testing-library/react@9.0.0 or higher
+// unmount and cleanup DOM after the test is finished.
+// eslint-disable-next-line no-undef
+afterEach(cleanup);
+
+describe('ChannelPreviewMessenger', () => {
+  let chatClient;
+  let userId;
+  let userToken;
+  let channelID;
+  let channel;
+  let channelName;
+  beforeEach(async function() {
+    chatClient = getTestClient();
+    userId = `thierry-${uuidv4()}`;
+    userToken = createUserToken(userId);
+    channelID = uuidv4();
+    chatClient.setUser(
+      {
+        id: userId,
+      },
+      userToken,
+    );
+    channelName = uuidv4();
+    channel = chatClient.channel('messaging', channelID, {
+      name: channelName,
+    });
+    await channel.create();
+    await channel.watch();
+  });
+
+  it('Channel name and empty message should be present in begining', () => {
+    const { getByText } = render(
+      <ChannelPreviewMessenger
+        latestMessage=""
+        channel={channel}
+        activeChannel={channel}
+        setActiveChannel={() => {}}
+      />,
+    );
+
+    const emptyMessageText = 'Nothing yet...';
+    expect(getByText(emptyMessageText)).toBeTruthy();
+    expect(getByText(channelName)).toBeTruthy();
+  });
+
+  it('Latest message should be present in cacatenated state', () => {
```

Comment:
typo aside, you should make test names more explicit; this test does 2 things but the description is not very clear about it